kin-sdk-core/src/main/java/kin/sdk/core/KinConsts.java

```diff
@@ -1,61 +1,38 @@
-
-    /**
-     * <p>SHA3 (Keccak-256) hash of ERC-20's Transfer event:</p>
-     * {@code Transfer(address,address,uint256)}
-     */
-    static final String TOPIC_EVENT_NAME_SHA3_TRANSFER = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef";
-
-    static long getTransferKinGasLimit(ServiceProvider provider) {
-        return NetworkConstants.fromProvider(provider).transferKinGasLimit;
-    }
+    static final String KIN_ASSET_CODE = "KIN";
```

Comment:
should file contacts file only include static final strings/init etc and not end, or static methods?